tests/test_solvers/test_kalman.py

```diff
@@ -40,6 +40,13 @@ def data(random):
         quasisep.CARMA.init(
             alpha=np.array([1.4, 2.3, 1.5]), beta=np.array([0.1, 0.5])
         ),
+        quasisep.CARMA.init(
```

Comment:
These tests aren't actually the right place to test this. The `KalmanSolver` isn't actually used in general. The place to demonstrate issues with the implementation is in [`test_solver`](https://github.com/dfm/tinygp/blob/992eb3db76587c62a27330835f91541b00acde8f/tests/test_solvers/test_quasisep/test_solver.py) because that's the solver that actually gets used!

Reply:
OK, I will remove them.